plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -151,11 +158,7 @@ class ContainerServiceFormSection extends Component {
   }
 
   getGPUSLabel() {
-    const {data, path} = this.props;
-    const typePath = this.getFieldPath(path, 'type');
-    const disabled = findNestedPropertyInObject(data, typePath) !== MESOS;
-
-    if (disabled) {
+    if (this.isGpusDisabled()) {
```

Comment:
Not that @jfurrow is working on something to make this easier: https://github.com/dcos/dcos-ui/pull/1867 (closed, but he will put up another PR).

This is not to block this PR, just to let you know 😉